tinker-actions/src/fde_dmv/enable_dmv_emt.sh

```diff
@@ -182,9 +182,12 @@ make_partition() {
 	swap_size=$(echo "$ram_size" | awk '{printf ("%.0f\n", sqrt($1))}')
     fi
 
-    swap_size=$(( swap_size * 1024 ))
+    swap_size=$(( swap_size ))
 
     total_size_disk=$(fdisk -l ${DEST_DISK} | grep -i ${DEST_DISK} | head -1 |  awk '/GiB/{ print int($3)*1024} /TiB/{ print int($3)*1024*1024}')
+    # To ensure perfect boundary alignment and avoid sector errors, 
+    # we have reduced the total disk size by 1 MB for the last partition.
+    total_size_disk=$(( total_size_disk - 1))
```

Comment:
This doesnt make sense to me. what is that you are trying to achieve here? 
i havent seen any such issues you mentioned in the comments till now.
if this is to make it work on VEN?